rdmo/tasks/managers.py

```diff
@@ -15,6 +15,16 @@ class TaskQuestionSet(CurrentSiteQuerySetMixin, GroupsQuerySetMixin, Availabilit
     def filter_catalog(self, catalog):
         return self.filter(models.Q(catalogs=None) | models.Q(catalogs=catalog))
 
+    def filter_available_views_for_project(self, project):
```

Comment:
I think that a `Project` should be able to find the `views` that are available for itself. Do you agree and would this query be correct for that?
I've changed the `filter_group` a little bit to handle a list of `users` as well as a single `user`.

Reply:
this kind of method is also needed for an endpoint (for project.views) in the new Project Detail in React I think

Reply:
As discussed in the meeting.